src/button.main.module.css

```
@@ -0,0 +1,21 @@
+.button {
+  color: white;
+  background-color: darkslateblue;
+  border-radius: 10pt;
+  border: 0pt;
+  padding: 24pt 44pt;
+  margin: 0pt 5pt;
+  transition-duration: 0.3s;
+}
+
+.button:hover {
```

to avoid repeating yourself:

```suggestion
.button:hover,
.button:focus {
```